File: PRAKTIKUM 3/Bahasa C/PRAK304-2510817120014-NailaAtresiaRahma.c
#include <stdio.h>

int main() { 

    int x;
    scanf("%d", &x);

    if(x==0) {
        printf("Nol");
    } else if(x>0 && x<10) {
        printf("Satuan"); 
    } else if(x>10 && x<20) {
        printf("Belasan");
    } else if(x==10 || x>19 && x<100) {
        printf("Puluhan");
    } else if (x>99) {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    

    return 0;
}

File: PRAKTIKUM 3/Bahasa Python/PRAK304-2510817120014-NailaAtresiaRahma.py
x = int(input())

if x==0:
    print("Nol")
elif x>0 and x<10:
    print("Satuan")
elif x>10 and x<20:
    print("Belasan")
elif x==10 or x>19 and x<100:
    print("Puluhan")
elif x>99:
    print("Anda Menginput Melebihi Limit Bilangan")